### PR TITLE
Feature: APP-918 - Execution Terminal Status

### PR DESCRIPTION
**Pull request by Fabricevladimir**
## Description
- Sets proper status for execution terminal based on the proposal status, wallet connection and voter eligibility status
- Refactors Execution Terminal to match new designs
- Adds empty state for Execution Terminal

**Execution Terminal is broken for main Withdraw flow as well**
Also, no action is being passed to Execution Terminal on proposal details page on purpose because we are waiting on SDK method for action decoding.

Task: [APP-918](https://aragonassociation.atlassian.net/browse/APP-918)

## Type of change

- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] I have selected the correct base branch.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] Any dependent changes have been merged and published in downstream modules.
- [ ] I ran all tests with success and extended them if necessary.
- [ ] I have updated the ``CHANGELOG.md`` file in the root folder.
- [x] I have tested my code on the test network.

### CODE REVIEW

File: packages/web-app/src/locales/en/common.json [hunk 1]
```diff
@@ -167,8 +167,7 @@
     "newTokens": "New Tokens",
     "newHolders": "New Holders",
     "totalTokens": "Total Tokens",
-    "totalHolders": "Total Holders",
-    "seeCommunity": "See Community"
```
Comment (WissenIstNacht) [inline packages/web-app/src/locales/en/common.json:171]: Removing duplicates, nice.

File: packages/web-app/src/locales/en/common.json [hunk 2]
```diff
@@ -218,7 +217,15 @@
     "noProposals": "No proposals found!",
     "executionCard": {
       "title": "Execution",
-      "description": "These smart actions are executed when the proposal reaches sufficient support. Find out which actions are executed."
+      "description": "These smart actions are executed when the proposal reaches sufficient support.",
+      "addAction": "Add Action",
```
Comment (WissenIstNacht) [inline packages/web-app/src/locales/en/common.json:221]: Adding duplicates, nice 😁

File: packages/web-app/src/pages/proposal.tsx [hunk 3]
```diff
@@ -145,110 +154,126 @@ const Proposal: React.FC = () => {
...
-            };
-          }
-          break;
+  const [
+    voteNowDisabled,
+    statusLabel,
+    alertMessage,
+    executionStatus,
+    handleVoteClicked,
+  ] = useMemo(() => {
```
Comment (WissenIstNacht) [inline packages/web-app/src/pages/proposal.tsx:163]: Just a note on... let me call them "large" memos. In other words, memo where you compute a couple of values and then return them them together as an object:

I understand that it makes it somewhat cleaner to have everything in on place. However, keep in mind that you now perform a huge computation (whether this actually and expensive computation is an other discussion but not relevant here) in one go. This means that whenever _any_ of the dependencies change, the whole block get re-executed. And obviously the likelyhood of this happening actually increases with every dependeny you (are forced) to add. This kind of defeats the purpose. If you break them up you might have situation where one changes and triggers a couple of others to change. But at least you might have some that were not affected.

//Edit: Just to be clear, sometimes things are what they are, and you have to. I've not analysed the situation in full here. Maybe it's necessary. I only wanted to put the thought out there :D

**Review by WissenIstNacht — APPROVED** [review]
Thanks :)

**Review by RakeshUP — APPROVED** [review]

File: packages/web-app/src/locales/en/common.json (hunk 2 above)
Comment (Fabricevladimir) [reply packages/web-app/src/locales/en/common.json:221]: It should be just a modification I think lol

File: packages/web-app/src/pages/proposal.tsx (hunk 3 above)
Comment (Fabricevladimir) [reply packages/web-app/src/pages/proposal.tsx:163]: Fair observation. I would like to point out thought that in this case, nothing should change save for the login state/wrong network status. Plus, the meat of the memo are actually small `cases` of the `switch`.

**Pull request merged**